themes/agnoster.zsh-theme

```diff
@@ -198,6 +198,19 @@ prompt_virtualenv() {
   fi
 }
 
+# Anaconda: current working conda env
+# To replace the conda modification of PS1 do `conda config --set changeps1 False`
+# or add 'changeps1: False' to your .condarc file.
+prompt_conda() {
+  local conda_env="$CONDA_DEFAULT_ENV"
+  if [[ -n $conda_env ]]; then
+    local conda_changeps1="`conda config --show changeps1`" 
+    if [[ $conda_changeps1 == "changeps1: False" && $conda_env != "base" ]]; then
+      prompt_segment blue black "($CONDA_DEFAULT_ENV)"
```

Comment:
Well, I know that the convention for `venv` is to wrap the env name in `()`, but - why? it doesn't add anything to the user.
May I suggest:
```suggestion
      prompt_segment blue black "conda:$conda_env"
```

This will let the user know that they are in a conda env name `$conda_env`.

Attached is an example
![image](https://user-images.githubusercontent.com/831127/76022953-d67ff380-5f30-11ea-870c-f3008c8a3834.png)

Reply:
Thank you for your suggestions on performance and formatting. The `()` was to be consistent with conda but I agree that your formatting is much more clear.

Reply:
Sure thing. I hope your PR will be accepted soon. I actually thought that this PR is abandoned because it was opened since Aug 2018...

The format works for me, because I use both venv and conda for different tasks. I never understood those () around the env name anyway...